src/renderer/components/+config-autoscalers/hpa-details.tsx

```diff
@@ -57,47 +60,46 @@ class NonInjectedHpaDetails extends React.Component<HpaDetailsProps & Dependenci
     const { object: hpa } = this.props;
 
     const renderName = (metric: HorizontalPodAutoscalerMetricSpec) => {
+      const metricName = getMetricName(metric);
+
       switch (metric.type) {
         case HpaMetricType.ContainerResource:
 
         // fallthrough
         case HpaMetricType.Resource: {
           const metricSpec = metric.resource ?? metric.containerResource;
-          const addition = metricSpec.targetAverageUtilization
-            ? " (as a percentage of request)"
-            : "";
```

Comment:
Removed this because we already show percent symbol if `averageUtilization` used. And its easier for new separated typings for v2 and v2beta1 specs.